src/System.Private.CoreLib/shared/System/Numerics/Vector.tt

```diff
@@ -75,10 +73,7 @@ namespace System.Numerics
         public static Vector<T> Zero
         {
             [Intrinsic]
-            get
-            {
-                return s_zero;
-            }
+            get => s_zero;
```

Comment:
Some of these changes don't look to be reflected in `Vector.cs` because they were already made there, but not here. Is that correct?

Reply:
Yes, I have back-ported changes from Vector.cs to Vector.tt to make the template current.